locals.tf

```diff
@@ -15,11 +15,15 @@ locals {
   cluster_endpoint_private_access_cidrs = var.cluster_endpoint_private_access_cidrs == null ? distinct(concat(module.vpc.public_subnet_cidrs, module.vpc.private_subnet_cidrs)) : var.cluster_endpoint_private_access_cidrs
   postgres_public_access_cidrs          = var.postgres_public_access_cidrs == null ? local.default_public_access_cidrs : var.postgres_public_access_cidrs
 
+  # IPs
+  create_jump_public_ip                 = var.create_jump_public_ip == null ? true : var.create_jump_public_ip
+  create_nfs_public_ip                  = var.create_nfs_public_ip == null ? false : var.create_nfs_public_ip
```

Comment:
Very confused here. These locals were removed from the Azure and the variable defaults were added to the variables.tf file and  yet they are now added here. Also the GCP code base sets these defaults as well. Need to keep consistency across the providers for our sanity. Please set the default correctly and remove these locals or update the other 2 repos to reflect the need for the locals. The items needed for the variables.tf are listed below.

```terraform
variable "create_nfs_public_ip" {
  default = false
  type = bool
}

variable "create_jump_public_ip" {
  default = true
  type = bool
}
```